src/content/documentation/publish/distribute-manifest-versions.md

```diff
@@ -0,0 +1,123 @@
+
+{% endcapture %}
+{% include modules/column-w-toc.liquid,
+  id: "transfer-ownership"
+  content: content_with_toc
+%}
+
+<!-- END: Content with Table of Contents -->
+<!-- Single Column Body Module -->
+
+{% capture content %}
+
+## Prepare your MV3 version
+
+To support automatic updates to newer MV3 versions, you need to create and make available an update manifest file and include the file's location in the extension's manifest file.
+
+See [Updating your extension](/documentation/manage/updating-your-extension/) for details about the update manifest file format and manifest change needed.
+
+{% endcapture %}
+{% include modules/one-column.liquid,
+  id: "listing-on-amo"
+  content: content
+%}
+
+<!-- END: Single Column Body Module -->
+<!-- Single Column Body Module -->
+
+{% capture content %}
+
+## Get a signed MV3 version
+
+To get a signed version of your MV3 extension:
+
+1. On AMO, when you upload the MV3 extension version, change “Where to host version”  to “on your own”.
+   ::: note
+   You need to change it back to “on this site” for MV2 versions, as the most recently used channel is remembered.
+   :::
+2. For API uploads, specify `channel=unlisted` for the self-distribution channel.
+
+See [Self distribution](/documentation/publish/submitting-an-add-on/#self-distribution) for more details on the process.
+
+{% endcapture %}
+{% include modules/one-column.liquid,
+  id: "listing-on-amo"
+  content: content
+%}
+
+<!-- END: Single Column Body Module -->
+<!-- Single Column Body Module -->
+
+{% capture content %}
+
+## How to distribute the MV3 version
+
+Self-distributed versions cannot be downloaded by users directly from AMO. You must distribute the experimental XPI to users yourself, such as by a download from your website, email, etc.
+
+See [Distributing an add-on yourself](/documentation/publish/self-distribution/) for more details on the process.
+
+{% endcapture %}
+{% include modules/one-column.liquid,
+  id: "self-distribution"
+  content: content
+%}
+
+<!-- END: Single Column Body Module -->
+<!-- Single Column Body Module -->
+
+{% capture content %}
+
+## Transition your experimental MV3 version to your live version
+
+When you want to support just MV3-compatible versions of Firefox for your extension, visit AMO and upload the MV3 version to the primary AMO listed channel (“on this site”), without the `update_url` in the [`browser_specific_settings`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/browser_specific_settings) manifest key. If you are distributing the MV3 version from your website also specify [`install_origins`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/install_origins) in the manifest. Then, when signed, download the XPI and distribute it to your users as you did originally. The lack of the `update_url` manifest key means users will get updates from the AMO channel in the future.
```

Comment:
Why is `install_origins` still mentioned here?

Reply:
I think if we just drop "If you are distributing the MV3 version from your website also specify [`install_origins`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/manifest.json/install_origins) in the manifest. " the rest makes sense.